src/haz3lweb/view/CtxInspector.re

```diff
@@ -3,21 +3,27 @@ open Node;
 open Util.Web;
 
 let context_entry_view =
-    ((name: string, {typ, _}: Haz3lcore.Ctx.entry)): Node.t =>
+    (
+      (
+        name: string,
+        {item: typ, _}: Haz3lcore.TermCtx.entry(Haz3lcore.Typ.t),
+      ),
+    )
+    : Node.t =>
   div(
     ~attr=clss(["context-entry"]),
     [text(name), text(":"), Type.view(typ)],
   );
 
 let ctxc = "context-entries";
 
-let exp_ctx_view = (ctx: Haz3lcore.Ctx.t): Node.t =>
+let exp_ctx_view = (ctx: Haz3lcore.TermCtx.t): Node.t =>
   div(
```

Comment:
something is broken with either the context itself or the view code -- entries are showing up doubled in the context inspector

Reply:
Looking into it...